projects/rescuechi/petfinder/data_cleaner.py

```diff
@@ -60,11 +60,38 @@ def explode_column(col, col_prefix) -> pd.DataFrame:
 
     return exploded
 
+def find_org(org_id, org_name) -> pd.DataFrame:
```

Comment:
Nice, this is a really cool implementation of mapping together the organization IDs to the organization names. One snazzy thing about pandas is that it has a lot of this dataframe manipulation already implemented -- for example, I think that the `join` method ([link](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.join.html)) essentially does what you've implemented here.